### PR TITLE
Add a script to check for @watch in tests

### PR DESCRIPTION
**Pull request by bitdivision**
### What is the context of this PR?
I always forget to remove @watch statements from functional tests.

### How to review 
Add a watch statement and make sure the lint script fails.

### CODE REVIEW

File: scripts/check_watch_statements.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if grep -r "@watch" tests/functional/spec; then
```
Comment (MebinAbraham) [inline scripts/check_watch_statements.sh]: May as well check for `.only` and `@focus` as well?

Comment (bitdivision) [reply scripts/check_watch_statements.sh]: Done

File: scripts/run_lint.sh [hunk 2]
```diff
@@ -18,6 +18,9 @@ function display_result {
   fi
 }
 
+./scripts/check_watch_statements.sh
+display_result $? 1 "Functional tests watch check"
```
Comment (MebinAbraham) [inline scripts/run_lint.sh]: Not just watch now.

**Review by MebinAbraham — APPROVED** [review]

File: scripts/check_watch_statements.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+if grep -Er "@watch|@focus|\.only" tests/functional/spec; then
```
Comment (MebinAbraham) [inline scripts/check_watch_statements.sh]: File name prob should be changed also. lint_spec_files?

File: scripts/run_lint.sh (hunk 2 above)
Comment (bitdivision) [reply scripts/run_lint.sh]: Yeah, but I don't have a better name, and they all acheive a watch, so seems reasonable to me?

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**